deployer/templates/fluentd.yaml

```diff
@@ -159,6 +159,10 @@ objects:
               value: ${USE_MUX_CLIENT}
             - name: "MUX_ALLOW_EXTERNAL"
               value: ${MUX_ALLOW_EXTERNAL}
+            - name: "BUFFER_QUEUE_LIMIT"
```

Comment:
OK. @jcantrill let's update the template for now, removing it will be on the TODO but not something that's already finished so no use anticipating it here :)